react-components/src/components/Architecture/useClickOutside.tsx

```diff
@@ -1,24 +0,0 @@
-/*!
```

Comment:
Is click outside now handled by cogs library?

Reply:
Yes, it is :+1: